src/wp-includes/ms-functions.php

```diff
@@ -1118,6 +1118,21 @@ function wpmu_activate_signup($key) {
 	return array('blog_id' => $blog_id, 'user_id' => $user_id, 'password' => $password, 'title' => $signup->title, 'meta' => $meta);
 }
 
+/**
+ * Deletes am associated signup entry when a user is deleted from the database.
```

Comment:
This typo can stay for now, and if WP ever fixes it then we can backport the fix. This should be an easy way to get contributor credits over there also.

Reply:
I have reported this back upstream https://core.trac.wordpress.org/ticket/52628#comment:24